dev-util/hyprwayland-scanner/hyprwayland-scanner-0.2.0.ebuild

```
@@ -0,0 +1,25 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A Hyprland implementation of wayland-scanner, in and for C++"
+
+HOMEPAGE="https://github.com/hyprwm/hyprwayland-scanner/"
+
+inherit cmake
```

inherit goes before DESCRIPTION, also remove newline between DESCRIPTION & HOMEPAGE

```suggestion
EAPI=8

inherit cmake

DESCRIPTION="A Hyprland implementation of wayland-scanner, in and for C++"
HOMEPAGE="https://github.com/hyprwm/hyprwayland-scanner/"
```